data-types.md

```diff
@@ -294,18 +294,18 @@ Other special group types may be added in future MDS releases as relevant agency
 [Top][toc]
 
 [costs-and-currencies]: /general-information.md#costs-and-currencies
-[event-times]: /general-information.md#event-times
+[event-times]: #event-times
 [gbfs-station-info]: https://github.com/NABSA/gbfs/blob/master/gbfs.md#station_informationjson
 [gbfs-station-status]: https://github.com/NABSA/gbfs/blob/master/gbfs.md#station_statusjson
 [geography-driven-events]: /general-information.md#geography-driven-events
 [gps]: #gps-data
 [iso4217]: https://en.wikipedia.org/wiki/ISO_4217#Active_codes
 [modes]: /modes/README.md
-[propulsion-types]: /general-information.md#propulsion-types
-[stop-based-geo]: #stop-based-geographic-data
+[propulsion-types]: #propulsion-types
+[stop-based-geo]: /general-information.md#stop-based-geographic-data
```

Comment:
I think this should link to /general-information.md#geographic-data as there is no anchor on the page for #stop-based-geographic-data

Stop Based Geographic data was removed in 2.0, in favor of mentioning it in Telemetry:  https://github.com/openmobilityfoundation/mobility-data-specification/blob/25a87400959eee89e2b06b694ef157a5fef8c138/data-types.md#telemetry

Reply:
stop-based-geo is unused once I repoint the reference on line 111 to Telemetry.